devel/distcc/Portfile

```diff
@@ -1,15 +1,37 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github 1.0
 
 name                distcc
-version             3.2rc1
-revision            1
+subport distcc-devel {}
+
+if {${subport} eq "distcc"} {
+    conflicts-append "distcc-devel"
+    github.setup    distcc distcc 3.3 v
+    checksums       rmd160  fa7b0112200aaf14d1b300a0c9105eb96129db1c \
+                    sha256  cb35875659e44becca4c78b04ae9b64f6abb40f75f80cafac61b5f48abb0c363
+    set PPREFIX     "./"
+} else {
+    conflicts-append "distcc"
+    # fetch from RJVB's fork which contains the patches to upstream
+    github.setup    RJVB distcc 145cdb4f9630af910f96d379bbde9cd0da376c28
+    fetch.type      git
```

Comment:
Why? It doesn't look like you're fetching submodules, which is the most common reason for fetching from git.